src/python_testing/conformance_support.py

```diff
@@ -294,46 +308,21 @@ def __str__(self):
         return ', '.join(op_strs)
 
 
-def parse_callable_from_xml(element: ElementTree.Element, params: ConformanceParseParameters) -> Callable:
-    if len(list(element)) == 0:
-        # no subchildren here, so this can only be mandatory, optional, provisional, deprecated, disallowed, feature or attribute
-        if element.tag == MANDATORY_CONFORM:
-            return mandatory()
-        elif element.tag == OPTIONAL_CONFORM:
-            return optional()
-        elif element.tag == PROVISIONAL_CONFORM:
-            return provisional()
-        elif element.tag == DEPRECATE_CONFORM:
-            return deprecated()
-        elif element.tag == DISALLOW_CONFORM:
-            return disallowed()
-        elif element.tag == FEATURE_TAG:
-            try:
-                return feature(params.feature_map[element.get('name')], element.get('name'))
-            except KeyError:
-                raise ConformanceException(f'Conformance specifies feature not in feature table: {element.get("name")}')
-        elif element.tag == ATTRIBUTE_TAG:
-            # Some command conformance tags are marked as attribute, so if this key isn't in attribute, try command
-            name = element.get('name')
-            if name in params.attribute_map:
-                return attribute(params.attribute_map[name], name)
-            elif name in params.command_map:
-                return command(params.command_map[name], name)
-            else:
-                raise ConformanceException(f'Conformance specifies attribute or command not in table: {name}')
-        elif element.tag == COMMAND_TAG:
-            return command(params.command_map[element.get('name')], element.get('name'))
-        elif element.tag == CONDITION_TAG and element.get('name').lower() == 'zigbee':
+def parse_basic_callable_from_xml(element: ElementTree.Element) -> Callable:
+    if len(list(element)) != 0:
```

Comment:
maybe instead of using len just `if list(element):` would work  (https://peps.python.org/pep-0008/#programming-recommendations says to use `if seq` and `if not seq` instead of using len)

a few more instances below.